extensions/amp-carousel/amp-carousel.md

```diff
@@ -153,6 +162,16 @@ corresponding carousel item displays.
 
 [/example]
 
+### Accessibility considerations for `amp-carousel`
```

Comment:
Note: There is also a v1 documentation file at `extensions/amp-carousel/0.1/amp-carousel.md` if it may require different considerations.

Reply:
Apologies, we missed that. Backported the advice to the 0.1 documentation as well, as it was still applicable.